test_vae_training/0_explore_data.py

```diff
@@ -0,0 +1,62 @@
+# ---
```

Comment:
FYI, the .ipynb version of this file isn't rendering in git. Not sure if it's a formatting issue or just git having problems. If it renders on your end there's probably no problem.

Reply:
Nevermind, it's working now, definitely a git issue

Reply:
In my experience, using https://nbviewer.jupyter.org/ has been easier than seeing whether github will decide to render a notebook